core/js/src/main/scala/org/log4s/Logger.scala

```diff
@@ -0,0 +1,10 @@
+package org.log4s
+
+case class Logger(name: String) {
```

Comment:
Is there no way to write to a console?

I still wonder whether the log4s project would be interested in cross-compilation.

Reply:
Yes we can write to console. This is only an interim to get things compiling